docs/source/conf.py

```diff
@@ -61,8 +70,28 @@
     'sphinx.ext.doctest',
     'sphinx_panels',
     'sphinxcontrib.images',
+    'nbsphinx',
 ]
 
+_GIT_COMMIT = "dev"  # TODO(ravi) -- replace this with the git commit
```

Comment:
This will be fixed after #1113, which includes code to determine the git commit.

Reply:
Done